experiments/browser_based_querying/src/TrustfallPlayground.tsx

```diff
@@ -274,6 +276,14 @@ export default function TrustfallPlayground(props: TrustfallPlaygroundProps): JS
             <div ref={resultsEditorRef} css={cssEditor} />
           </Paper>
         </Grid>
+        <Grid container item xs={4} direction="column" sx={{ flexWrap: 'nowrap' }}>
```

Comment:
This might not work as expected because all `item`s in a grid container should add up to 12.

Reply:
Oh interesting I assumed it was all based on proportions